app/src/main.js

```diff
@@ -12,8 +12,8 @@ import updateChecker from './modules/autoUpdater';
 import server from '../server';
 
 require('babel-polyfill'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Please user ES6 import instead of `require`.

Reply:
thanks for pointing this Yashar, I was doing some tests and I can confirm that is totally necessary, however I remove it from this file and put it in the correct place.